onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -35,6 +35,7 @@
 import org.onebusaway.android.io.elements.ObaRoute;
 import org.onebusaway.android.io.elements.ObaShape;
 import org.onebusaway.android.io.elements.ObaStop;
+import org.onebusaway.android.io.elements.ObaStopElement;
```

Comment:
@barbeau In ```BaseMapFragment.java ``` there are other unused variables: `CAMERA_MOVE_NOTIFY_THRESHOLD_MS`,  `mStopUserMap`, `mFocusStop`, `mFocusStopRoutes`. Do you want me to clean them up, or are we planning to use them in the future?